requirements.txt

```diff
@@ -2,8 +2,13 @@ autopep8==1.6.0
 fastapi==0.80.0
 gspread==5.4.0
 gunicorn==20.1.0
+numpy==1.23.2
 oauth2client==4.1.3
+openpyxl==3.0.10
```

Comment:
Is this used anywhere?

Reply:
This is downloaded with pandas which are used to open .xlsx and .csv.